code/modules/power/cable.dm

```diff
@@ -415,3 +412,172 @@
 		logTheThing(LOG_STATION, user, "lays a cable[powered == 1 ? " (powered when connected)" : ""] at [log_loc(src)].")
 
 	return
+
+/// a cable spawner which can spawn multiple cables to connect to other cables around it.
+/obj/cablespawner
+	level = 1
+	name = "power cable spawner"
+	desc = "An item that should spawn actual cables. If you're reading this, something's wrong."
+	icon = 'icons/obj/power_cond.dmi'
+	icon_state = "superstate"
+	var/iconmod = null
```

Comment:
Is this used anywhere?

Reply:
The `desc` variable isn't used actually, yeah, so i might clear that up. `iconmod` might be used when doing reinforced cable spawners and such, so i'm leaving that one in.

Reply:
turns out iconmod isn't used at all since it uses `new()` on the `cable_laying()` bit, so i'm clearing out `desc` and `iconmod`